manifests/0000_51_olm_00-olm-operator.yml

```diff
@@ -3,13 +3,11 @@ kind: OLM
 metadata:
   name: cluster
   annotations:
-    include.release.openshift.io/hypershift: "true"
     include.release.openshift.io/ibm-cloud-managed: "true"
```

Comment:
Do we need to remove this label as well?

Reply:
Need to remove the following:
```
include.release.openshift.io/hypershift: "true" // not a thing
include.release.openshift.io/ibm-cloud-managed: "true" // hypershift
include.release.openshift.io/single-node-developer: "true" // not a thing
```

Reply:
Agree, fixed it.